packages/fxa-event-broker/src/pubsub-proxy/pubsub-proxy.controller.spec.ts

```diff
@@ -171,53 +177,61 @@ describe('PubsubProxy Controller', () => {
       it(`notifies successfully on ${key}`, async () => {
         mockWebhook();
         const message = creatFunc();
-        const res = mockResponse();
-
-        await controller.proxy(
-          res as any,
-          {
-            message: { data: message, messageId: 'test-message' },
-            subscription: 'test-sub',
-          },
-          'abc1234'
-        );
-        expect(res.set).toBeCalledWith({ 'x-auth': 'Bearer ' + TEST_TOKEN });
+        expect.assertions(3);
+        try {
+          await controller.proxy(
+            {
+              message: { data: message, messageId: 'test-message' },
+              subscription: 'test-sub',
+            },
+            'abc1234'
+          );
+        } catch (err) {
+          const status = err.getStatus();
+          const body = err.getResponse();
+          expect(status).toBe(200);
+          expect(body).toStrictEqual({ token: 'Bearer ' + TEST_TOKEN });
```

Comment:
Yeah, this seems very strange to me

Reply:
It's a unit test, so the exception handling that transforms `HTTPException` classes into responses isn't firing. Thus these statements to verify that the code-path executed properly and resulted in the expected `HTTPException`.